docs/application/vstools/install.md

```diff
@@ -0,0 +1,170 @@
+2. Select **Install new Tizen SDK**.
+
+   ![Select new installation](media/howtoinstall-installwizard1.png)
+
+3. Read the license document and click **I Agree**.
+
+   ![Agree to license details](media/howtoinstall-installwizard2.png)
+
+4. Enter the root directory path where you want to install and click **Next**.
+
+   ![Set the installation path](media/howtoinstall-installwizard3.png)
+
+   The Tizen SDK installer is downloaded and the baseline SDK is installed automatically.
+
+   ![Installer download](media/howtoinstall-installwizard4.png)
+
+   ![Baseline SDK installation](media/howtoinstall-installwizard5.png)
+
+   Finally, the Tizen Package Manager installs the Tizen SDK tools.
+
+   ![Tool installation](media/howtoinstall-installwizard6.png)
+
+### Setting the Tizen Baseline SDK Path {#set-tizen-baseline-sdk}
+
+> **Note**  
+> If you downloaded the Visual Studio Tools for Tizen extension from the Visual Studio Marketplace and installed it on your computer, skip this step.
+
+
+You can use the Tizen Package Manager to set up the Tizen baseline SDK path or each tool path directly:
+
+- To set up the Tizen baseline SDK path:
+  1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
+  2. Select **Use installed Tizen SDK**.
+
+     ![Baseline SDK Install](media/howtoinstall-baselineinstall5.png)
+
+  3. Enter the root directory of your existing Tizen Studio installation.
+
+     ![Baseline SDK Install](media/howtoinstall-baselineinstall6.png)
+
+  > **Note**  
+  > If the installer gives a warning about your Tizen Studio version being too low, update the Tizen Studio by using the Tizen Package Manager after setting the tool path.
+
+- To set up each tool path directly:
+  1. In the Visual Studio IDE menu, go to **Tools &gt; Options &gt; Tizen &gt; Tools**.
+  2. Enter the root directory of your existing Tizen Studio installation in the **Tool Path** field.
+
+     ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
+
+     The other tools paths are automatically set up.
+
+### Installing Emulator Images
+
+If you do not have a real device, you can run applications in the Tizen emulator.
+
+To download emulator images, you can use the Tizen Package Manager or the Tizen Emulator Manager:
+
+- To use the Tizen Package Manager:
+  1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
+  2. Select the profiles and versions you want to install and click **Install**.
+
+     ![Package Manager](media/howtoinstall-packagemanager.png)
+
+- To use the Tizen Emulator Manager:
+
+  > **Note**  
+  > The Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
+
+  1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
+  2. Select the profiles and versions you want to install and click **OK**.
+
+     ![Emulator Manager](media/howtoinstall-emulatormanager.png)
```

Comment:
I think you forgot to upload updated howtoinstall-emulatormanager.png image. This version has a dot in it.

Reply:
done